client/Dockerfile

```diff
@@ -7,3 +7,5 @@ RUN npm install -g @vue/cli
 COPY package*.json ./
 
 RUN npm install
+
+RUN npm install vue-confetti --save
```

Comment:
Move this into `package.json`, that's where all packages that aren't needed to build the frontend go.